calc/calc.py

```diff
@@ -53,16 +50,36 @@ class Calc:
     7
 
     """
+    # Rabbit Hole:
+    #    You can read the position=0 syntax as saying "The default argument,
+    #    if you don't pass an argument to this function, is the value on the
+    #    right.
     def __init__(self, text, position=0, current_token=None):
-        """Constructor for a :class:`Calc` object."""
-        self.text = text
-        self.position = position
+        """Set's the text, position, and current_token attributes for the
```

Comment:
No need for an apostrophe in sets. Maybe use double backticks like in other functions so if rendered with Sphinx they'd look like code snippets?
```python
"""Sets the ``text``, ``position``, and ``current_token`` attributes...
"""
```